src/server/hotspot_partition_calculator.h

```diff
@@ -17,13 +17,17 @@
 
 #pragma once
 
-#include "hotspot_partition_stat.h"
 #include <gtest/gtest_prod.h>
+
 #include <dsn/perf_counter/perf_counter.h>
+#include <dsn/utility/flags.h>
+#include "hotspot_partition_stat.h"
 
 namespace pegasus {
 namespace server {
 
+DSN_DECLARE_int32(occurrence_threshold);
```

Comment:
why did you add `DSN_DECLARE_int32`? I think you can remove it. Add it in where you really use.